python/setup.py

```diff
@@ -150,7 +150,10 @@ def finalize_options(self):
 
   def run(self):
     mjpc_tasks_path = Path(__file__).parent.parent / "mjpc" / "tasks"
-    source_paths = tuple(mjpc_tasks_path.rglob("*.xml"))
+    source_paths = (
```

Comment:
what do these changes do? I don't think these changes affect the task?

Reply:
I believe the `.png` assets defining the cube textures don't get properly copied into the `python` version when running `setup.py` without this, though I don't think I made this change originally.

@vincekurtz do you remember if my description is accurate?

Reply:
Yup, that was me and that was the reason. Definitely not sure if this is the right solution, and it might be irrelevant with some assets loaded from mujoco menagerie now.

Reply:
@thowell I just tested the python build and we need to add pngs **and stls** to the source paths for an Allegro demo to work correctly, so I added that in (for some reason, the change isn't reflecting in the diff for me on github even though I can see the change in the latest commit).